### ISSUE TITLE
Breaking Changes since v1.0.2: Hide Pagination when single page is available (#3497)

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#3856]
Need to reimplement toggles in order-history template to complete ticket #3497 

Replace wherever occurs `order-history.component.html`:
```
            <cx-pagination
              [pagination]="orders.pagination"
              (viewPageEvent)="pageChange($event)"
            ></cx-pagination>
```
with
```

            <cx-pagination
              [pagination]="orders.pagination"
              [hideOnSinglePage]="true"
              (viewPageEvent)="pageChange($event)"
            ></cx-pagination>
```

Also create e2e tests for this

**Comment by tobi-or-not-tobi** [comment]
Fixed with 2.0

**Comment by Xymmer** [comment]
fixed in 2.0 with #6060
